datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JwsSigned.kt

```diff
@@ -14,13 +14,10 @@ import io.matthewnelson.encoding.core.Encoder.Companion.encodeToString
 data class JwsSigned(
     val header: JwsHeader,
     val payload: ByteArray,
-    val signature: CryptoSignature
+    val signature: CryptoSignature,
+    val plainSignatureInput: String,
```

Comment:
do we want to keep this private?

Reply:
No, because otherwise the `JwsService` from vclib can't access it